tests/conftest.py

```diff
@@ -142,6 +142,8 @@ def always_true(_: object) -> bool:
 @pytest.fixture(autouse=True)
 def can_send(monkeypatch, request) -> None:
     """Return always True for HostAgent.can_send()"""
+    if "no_fixture" in request.node.keywords:
+        return None
```

Comment:
What is the reason for this if-block? I did not understand it.

Reply:
For some of test cases I didn't want to use fixture, such as below, with this if block, I'm being able to skip fixtures:

```
    @pytest.mark.no_fixture
    def test_can_send(
        self,
    ) -> None:
        agent = HostAgent()
        agent._boot_pid = 12345
        with patch.object(os, "getpid", return_value=12344), patch(
            "instana.agent.host.HostAgent.handle_fork"
        ) as mock_handle, patch.dict("os.environ", {}, clear=True):
            agent.can_send()
            assert agent._boot_pid == 12344
            mock_handle.assert_called_once()

            with patch.object(agent.machine.fsm, "current", "wait4init"):
                assert agent.can_send() is True
```